framework/packages/abstract-sdk/src/apis/version_registry.rs

```diff
@@ -74,22 +75,31 @@ impl<'a, T: ModuleRegistryInterface> ModuleRegistry<'a, T> {
 
     /// Smart query for a module
     pub fn query_module(&self, module_info: ModuleInfo) -> AbstractSdkResult<Module> {
-        Ok(self.query_all_module_config(module_info)?.module)
+        Ok(self
+            .query_all_module_config(vec![module_info])?
```

Comment:
This `all` is weird, could we come up with a better name (and fix the doc-comment for it?)

`query_module_configs`?